### DISCUSSION TITLE
Relations

### DISCUSSION THREAD

**Discussion by newtec-foggia** [filamentphp/filament#13217] [Help]
### Package

Form builder

### Package Version

v3

### How can we help you?

How can I, when I am editing, view one or more relationships in the side panel?
![table with relations](https://github.com/filamentphp/filament/assets/83398776/5df4aa7c-993d-4634-b682-da0302f5391d)

**Comment by leandrocfe** [comment]
Are you using the panel builder?

Check the relation manager 

https://filamentphp.com/docs/3.x/panels/resources/relation-managers/#creating-a-relation-manager

**Reply by newtec-foggia** [reply]
Hi Leandro, the question is whether you can position the relationship tabs not at the bottom but laterally in a section
